src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
-body {
+@import "./utils/variables";
+
+@import "./blocks/card";
+@import "./blocks/stars";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
```

Comment:
Don't use * selector, unless you have a reason to.

In your case you can move this to `html` or `body`

PS: 👍  for resetting `box-sizing` for all elements